fix: auth result should be hashed

vkjr opened the pull request:
Fixing auth problem

vkjr committed:
fix: auth result should be hashed

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (16)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | eb9542ca | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/1/) | 2025-10-21 13:23:43 | ~4 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/1/consoleText) |
| :heavy_check_mark: | eb9542ca | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/1/) | 2025-10-21 13:27:38 | ~8 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251021-131923-eb9542-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251021-131923-eb9542-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | eb9542ca | [#1](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/1/) | 2025-10-21 13:28:20 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251021-131925-eb9542-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251021-131925-eb9542-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | eb9542ca | [#1](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/1/) | 2025-10-21 13:43:43 | ~24 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/WqZZjy) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FWqZZjy&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :x: | 16098bf5 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/2/) | 2025-10-22 11:19:55 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/2/consoleText) |
| :heavy_check_mark: | 16098bf5 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/2/) | 2025-10-22 11:24:29 | ~7 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251022-111726-16098b-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251022-111726-16098b-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16098bf5 | [#2](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/2/) | 2025-10-22 11:24:56 | ~7 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251022-111731-16098b-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251022-111731-16098b-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | 16098bf5 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/3/) | 2025-10-22 14:38:25 | ~2 hr 28 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/wNJTtx) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FwNJTtx&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :x: | c0784a87 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/3/) | 2025-10-22 15:17:22 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/3/consoleText) |
| :heavy_check_mark: | c0784a87 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/3/) | 2025-10-22 15:21:46 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251022-151452-c0784a-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251022-151452-c0784a-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | c0784a87 | [#3](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/3/) | 2025-10-22 15:22:23 | ~7 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251022-151458-c0784a-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251022-151458-c0784a-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | c0784a87 | [#4](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/4/) | 2025-10-22 17:52:15 | ~2 hr 37 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/RU39ur) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FRU39ur&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :heavy_check_mark: | dbc1993c | [#4](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/4/) | 2025-10-23 06:21:11 | ~9 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-061208-dbc199-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-061208-dbc199-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | dbc1993c | [#5](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/5/) | 2025-10-23 06:21:12 | ~8 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/6y2ruW) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2F6y2ruW&size=400x400&errorCorrectionLevel=L)|
| :x: | dbc1993c | [#4](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/4/) | 2025-10-23 06:22:28 | ~10 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/4/consoleText) |
| :heavy_check_mark: | dbc1993c | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/5/) | 2025-10-23 10:04:47 | ~5 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-095908-dbc199-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-095908-dbc199-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :x: | f711688a | [#5](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/5/) | 2025-10-23 12:15:24 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/5/consoleText) |
| :heavy_check_mark: | f711688a | [#5](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/5/) | 2025-10-23 12:20:13 | ~7 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-121242-f71168-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-121242-f71168-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | f711688a | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/6/) | 2025-10-23 12:20:58 | ~8 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-121246-f71168-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-121246-f71168-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | f711688a | [#6](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/6/) | 2025-10-23 12:29:28 | ~16 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/EPf5Tx) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FEPf5Tx&size=400x400&errorCorrectionLevel=L)|
| | | | | | | |
| :x: | c2837d75 | [#6](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/6/) | 2025-10-23 13:23:09 | ~2 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-22651/6/consoleText) |
| :heavy_check_mark: | c2837d75 | [#6](https://ci.status.im/job/status-mobile/job/prs/job/android-e2e/job/PR-22651/6/) | 2025-10-23 13:27:33 | ~6 min | `android-e2e` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-132056-c2837d-pr22651-x86_64.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-132056-c2837d-pr22651-x86_64.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | c2837d75 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/android/job/PR-22651/7/) | 2025-10-23 13:27:54 | ~6 min | `android` | [:robot:`apk`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-251023-132058-c2837d-pr22651-arm64-v8a.apk) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fstatus-im-mobile-prs.ams3.cdn.digitaloceanspaces.com%2FStatusIm-Mobile-251023-132058-c2837d-pr22651-arm64-v8a.apk&size=400x400&errorCorrectionLevel=L)|
| :heavy_check_mark: | c2837d75 | [#7](https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-22651/7/) | 2025-10-23 13:32:07 | ~11 min | `ios` | [:iphone:`ipa`](https://i.diawi.com/Eir4PQ) [:calling:](https://quickchart.io/qr?text=https%3A%2F%2Fi.diawi.com%2FEir4PQ&size=400x400&errorCorrectionLevel=L)|

vkjr committed:
fix: logging

vkjr committed:
fix: "save" instead of "share" on Android

siddarthkay committed:
shell: fix clojure shell when system has yarn v4

make run-clojure used to break when system had yarn v4 because yarn 1.22.22 would check for package.json in parent directories,
setting SKIP_YARN_COREPACK_CHECK prevents that.

vkjr committed:
fix: more logging

vkjr committed:
fix: qr logs
